exp-models/addon/models/account.js

```diff
@@ -23,5 +22,10 @@ export default DS.Model.extend(JamModel, {
         };
 
         return profiles.filter(getProfile)[0];
+    },
+    pastSessionsFor(experiment) {
+        return this.get('store').queryRecord(experiment.get('sessionCollectionId'), {
+            q: `profileId:${this.get('id')}*`
```

Comment:
I'm updating this to support a second optional parameter `profile`. If `profile` is unspecified it will use the wildcard, otherwise the query will be profile specific.